mlfinlab/labeling/fixed_time_horizon.py

```diff
@@ -0,0 +1,52 @@
+"""
+Chapter 3.2 Fixed-Time Horizon Method
+Described in "Classification-based Financial Markets Prediction using Deep Neural Networks" Dixon et al. (2016)
+"""
+
+import warnings
+import numpy as np
+
+
+def fixed_time_horizon(close, threshold, look_forward=1, standardized=False, window=None):
+    """
+    Fixed-Time Horizon Labelling Method
+    Returns 1 if return at h-th bar after t_0 is greater than the threshold, -1 if less, and 0 if in between
+
```

Comment:
Please separate the name of the method and description with a blank line. Also, please reference the **Advances in Financial Machine Learning** book and the page/chapter where the method can be found.